src/els_client.erl

```diff
@@ -29,14 +38,7 @@
         , workspace_symbol/1
         ]).
 
-%% gen_server callbacks
--export([ init/1
-        , handle_call/3
-        , handle_cast/2
-        , handle_info/2
-        , terminate/2
-        , code_change/3
-        ]).
+-export([ handle_responses/1 ]).
```

Comment:
This is the callback which is used by the specific implementation when a message is available.